lib/main.dart

```diff
@@ -1,17 +1,34 @@
+// ignore_for_file: public_member_api_docs
+
+/*
+Copyright 2022 by Mustafa Sezer <mustafa.sezer@hit-solutions.de>
+
+This file is part of picos.
+
+picos is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+Y; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.picos is distributed in the hope that it will be useful, but WITHOUT ANY WARRANT
+You should have received a copy of the GNU General Public License along with picos. If not, see <https://www.gnu.org/licenses/>.
+*/
+
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const Picos());
```

Comment:
I think this is a really unfortunate class name here, please use something more self-explainatory like MainAppScreen() or something you feel fits better.